python/lsst/analysis/tools/analysisParts/shapeSizeFractional.py

```diff
@@ -59,7 +70,7 @@ def __call__(self, data: KeyedData, **kwargs) -> KeyedData:
         return super().__call__(data, **kwargs | dict(mask=mask))
 
 
-class ShapeSizeFractionalProcess(KeyedDataAction):
+class ShapeSizeFractionalProcessLegacy(KeyedDataAction):
```

Comment:
Thank you for changing the name here to include "Legacy" since it is no longer used elsewhere.